pkg/runtime/reconciler.go

```diff
@@ -183,19 +187,23 @@ func (r *resourceReconciler) Sync(
 	rm acktypes.AWSResourceManager,
 	desired acktypes.AWSResource,
 ) error {
+	var err error
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("r.Sync")
+	defer exit(err)
+
 	var latest acktypes.AWSResource // the newly created or mutated resource
 
 	isAdopted := IsAdopted(desired)
-	log := ackrtlog.AdaptResource(
-		r.log, desired,
-		"is_adopted", isAdopted,
-	)
+	rlog.WithValues("is_adopted", isAdopted)
```

Comment:
why not info or debug so that we know which resource is emitting the log

also, can't see this log in sample log in description

Reply:
@surajkota `WithValues` isn't a log message. What it does is adapt the logger with one or more pairs of key-values that log messages after the call to `WithValues` will inject in their log lines.